src/ms-review.tex

```diff
@@ -510,9 +510,7 @@ \section{Gamma-ray Data Analysis}
   migration, $\mu=\frac{E}{E_{\rm true}}$.
 \end{itemize}
```

Comment:
wasn't there a part here, where we said that those counts are put into a Poissonian likelihood that is then minimised to obtian the $\hat{\theta}}$ parameters? Otherwise it's not clear what do we do with the predicted counts. I remember writing this small paragraph.